src/Components/CreateImageWizard/formComponents/Packages.js

```diff
@@ -78,24 +78,29 @@ const Packages = ({ defaultArch, ...props }) => {
         };
     });
 
-    const sortPackages = useCallback((packageList) => {
-        const sortResults = packageList.sort(searchResultsComparator(packagesSearchName.current));
+    const setPackagesAvailableSorted = (packageList, filter = filterAvailable) => {
```

Comment:
I had to add an optional filter parameter here since `filterAvailable` is now a state.